proto/zetachain/zetacore/crosschain/cross_chain_tx.proto

```diff
@@ -135,4 +135,5 @@ message CrossChainTx {
   repeated OutboundParams outbound_params = 10;
   ProtocolContractVersion protocol_contract_version = 11;
   RevertOptions revert_options = 12 [ (gogoproto.nullable) = false ];
+  string error = 13;
```

Comment:
Should be in the status object
We should also add a comment to explain the field (we should do it for all field in the object actually)

Reply:
Moved.